lib/LibeventReactor.php

```diff
@@ -60,7 +60,7 @@ public function run(callable $onStart = null) {
         }
 
         while ($this->isRunning) {
-            if ($this->immediates && !$this->doImmediates()) {
+            if (!$this->doImmediates()) {
```

Comment:
Why did you strip the boolean check here? The func doesnt do anything when immediates are empty?